src/components/adaptive-cards/AdaptiveCard/AdaptiveCard.stories.js

```diff
@@ -1860,6 +1860,39 @@ const exampleActionOpenUrl = {
   ],
 };
 
+const exampleActionShowCard = {
+  $schema: 'http://adaptivecards.io/schemas/adaptive-card.json',
+  type: 'AdaptiveCard',
+  version: '1.0',
+  body: [
+    {
+      type: 'TextBlock',
+      text: 'This card\'s action will show another card',
+    },
+  ],
+  actions: [
+    {
+      type: 'Action.ShowCard',
+      title: 'Action.ShowCard',
```

Comment:
let's use 'Show another card' so that it's clear that `title` does not have to have the same value as `type`